src/Parser/Monk/Brewmaster/Modules/Spells/IronSkinBrew.js

```diff
@@ -80,6 +147,30 @@ class IronSkinBrew extends Analyzer {
       });
   }
 
+  get suggestionThreshold() {
```

Comment:
Please update the original suggestion to use these values so that you don't have to keep the two values in sync.

Reply:
When you have 2 `suggestionThresholds` in one class please consider prefixing both with a name so they can be identified in isolation.